test/framework/src/main/java/org/elasticsearch/test/AbstractXContentTestCase.java

```diff
@@ -65,7 +62,8 @@
             T parsed = parseFunction.apply(parser);
             assertEqualsConsumer.accept(testInstance, parsed);
             if (assertToXContentEquivalence) {
-                assertToXContentEquivalent(shuffled, XContentHelper.toXContent(parsed, xContentType, false), xContentType);
+                assertToXContentEquivalent(shuffled, XContentHelper.toXContent(parsed, xContentType, toXContentParams,false),
```

Comment:
missing space in `toXContentParams,false` between the two arguments, here and anywhere else.